README.md

```diff
@@ -130,7 +140,7 @@ _Compile AREG SDK sources and examples:_
 |**Linux**| Open gnome-terminal in Linux and call “_make_” to compile with POSIX API.|
 
 > 💡 Compilation with _Eclipse_ under Windows might require to switch the Toolchain. For example, `Cygwin GCC`.<br>
-> 💡 For Linux the default compiler is `g++`. Set preferred C++17 compiler in [conf/make/user.mk](./conf/make/user.mk) file.
+> 💡 For Linux the default compiler is `g++`. Set preferred C++17 compiler in [conf/cmake/pre-project.cmake](./conf/cmake/pre-project.cmake) file.
```

Comment:
Can we please move all configuration specific settings in `user.cmake`?
As well, the `user.mk` is relevant for the `make`. Let's keep this. Instead, we can add separate sub-chapters (headlines) building with different tools. You can add texts and i can format it. Anyway there are other documents to update.

Reply:
Unfortunately @aregtech , this set of configuration which right now only consist of setting  compiler directory; **must** be included *before* calling to `project` and by putting it inside of `user.cmak` or `common.cmak` it'll break multiple configuration which needs to be included *after* `project`.

As for the change, I'll revert it then for now.

Reply:
@Ali-Nasrolahi , OK. Understood. But let's also think how to find a solution. Or at least whether it makes sense to think about solution :)